styles.css

```diff
@@ -0,0 +1,197 @@
+nav {
+  position: relative;
+  z-index: 99 !important;
+  background-color: #fff;
+}
+
+.nav1 {
+  width: 97%;
+  margin: auto;
+  background-color: #f72210;
+  color: #fff;
+}
+
+.nav1 p {
+  margin: auto 30px;
```

Comment:
- [ ] Use bootstrap for this, [learn more](https://getbootstrap.com/docs/4.0/utilities/spacing/)